File: VitorBasso11611BCC034/programas tarefa 1/java/micro/micro04.java
public class micro04{
    public static void main(String[] args){
        int x, num, intervalo;
        intervalo = 0;
        for (x = 1; x <= 5; x++){
            System.out.print("Digite um numero: ");
            num = Integer.parseInt(System.console().readLine());
            if(num >= 10){
                if(num <=150){
                    intervalo = intervalo + 1;
                }
            }
        }
        System.out.println("Ao total, foram digitados " + intervalo + " numeros no intervalo entre 10 e 150");
    }
}

File: trabalho 4/codigos/lua/micro/micro04.lua
function int main()
    int x, num, intervalo
    intervalo = 0
    for x = 1, 5, 1
    do
        print("Digite um numero: ")
        num = io.read('*n')
        if num >= 10 then
            if num <= 150 then
                intervalo = intervalo + 1
            end
        end
    end
    print("Ao total, foram digitados ")
    print(intervalo)
    print(" numeros no intervalo entre 10 e 150")
    return 1
end

main()
